src/main/java/com/epam/esm/entity/Order.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.esm.entity;
+
+import com.epam.esm.converter.ZonedDateTimeAttributeConverter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+@Entity(name = "Order")
+@Table(name = "orders")
+@JsonIgnoreProperties({"user"})
```

Comment:
Why `JsonIgnoreProperties` here?

Reply:
the problem no longer exists since we removed user's orders from User entity